hw/bitstream/BUILD

```diff
@@ -89,6 +89,7 @@ filegroup(
 
 filegroup(
     name = "otp_mmi",
+    testonly = True,
```

Comment:
I think `:rom_mmi` needs this, too.

Reply:
I'm happy to add it, but I'm not seeing it in my tests. I'll try not to worry about it.